AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -117,6 +117,9 @@ smythstoys.com#$#body { overflow: visible!important; }
 ||moneysavingexpert.com/cookie-consent-v*/msmCookieConsent.js
 reifendirekt.de###cookie-diss
 hagebau.de##.js-cookie-hint
+atnews.gr#$#.cli-modal-backdrop { display: none !important; }
+atnews.gr#$##cookie-law-info-bar { display: none !important; }
+atnews.gr#$#body { overflow: visible !important; }
```

Comment:
`atnews.gr#@##cookie-law-info-bar` please add to cookie_whitelist

Reply:
`#cookie-law-info-again` also block this element

Reply:
#cookie-law-info-again Please remove this element, it appears after accepting cookies.